src/components/CreateTeam/CreateTeamForm/container.js

```diff
@@ -48,31 +42,15 @@ export default Wrapped =>
         );
     }
 
-    handleFormStatus(name, value, formIsValid) {
-      const updatedFormData = { ...this.state.formData };
-      updatedFormData[name] = value;
-      this.setState({
-        formData: updatedFormData,
-        formIsValid,
-      });
-    }
-
-    handleFormSubmit = event => {
-      event.preventDefault();
-      this.setState({ formData: { ...this.state.formData, teamName: '' } });
-      this.props.onSuccess(this.state.formData);
+    handleFormSubmit = data => {
+      this.props.onSuccess(data);
     };
 
     render() {
       return (
         <Wrapped
           clients={this.state.clients}
-          formData={this.state.formData}
-          formIsValid={this.state.formIsValid}
-          formStatus={(name, value, formIsValid) =>
-            this.handleFormStatus(name, value, formIsValid)
-          }
-          submitForm={e => this.handleFormSubmit(e)}
+          handleFormSubmit={e => this.handleFormSubmit(e)}
```

Comment:
I don't think we get a dom event here so I'd change the `e` property name to something more descriptive.

Reply:
Nice spot. Yeah, it should be data and not e for event